gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -91,6 +92,7 @@
   static final String RENEW_PATH = "/renew";
   static final String REVOKE_PATH = "/revoke";
   private static final String TARGET_ENDPOINT_PULIC_CERT_PEM = "knox.token.target.endpoint.cert.pem";
+  private static final long MILLIS_IN_DAY = 86400000L;
```

Comment:
nit: TimeUnit.DAYS.toMillis()

Reply:
Nice - can follow up in another commit. I have other changes in mind as well anyway.